src/main/java/uk/gov/hmcts/darts/common/datamanagement/api/DataManagementFacadeImpl.java

```diff
@@ -155,6 +167,22 @@ private DownloadResponseMetaData getDataFromStorage(List<ExternalObjectDirectory
         throw new FileNotDownloadedException(logBuilder.toString());
     }
 
+    private void createUnstructuredData(
+        DownloadResponseMetaData downloadResponseMetaData,
+        ExternalObjectDirectoryEntity eodEntityToDelete
+    ) throws IOException {
+        if (downloadResponseMetaData.getInputStream() != null) {
+            BinaryData binaryData = BinaryData.fromStream(downloadResponseMetaData.getInputStream());
+            CompletableFuture<Void> createUnstructuredJob = CompletableFuture.runAsync(() -> {
+                unstructuredDataHelper.createUnstructured(
+                    eodEntityToDelete,
+                    downloadResponseMetaData.getEodEntity(),
+                    binaryData);
+            });
+            UnstructuredDataHelper.addToJobsList(createUnstructuredJob);
```

Comment:
Calling a static method here is a little contrary to the general IoC approach we're taking elsewhere. Can we instead autowire the `unstructuredDataHelper` component, and call it's `addToJobsList()` method in a non-static way?

Reply:
removed statics